core/src/main/java/ai/timefold/solver/core/impl/constructionheuristic/DefaultConstructionHeuristicPhase.java

```diff
@@ -45,7 +48,12 @@ public TerminationStatus getTerminationStatus() {
 
     @Override
     public String getPhaseTypeString() {
```

Comment:
Since `Phase` is not public API, we have some freedom here.
If you decide to keep `PhaseType` (see above), maybe we return that here, instead of the String? If we ever need to exhaustively switch over phase types, this will help.